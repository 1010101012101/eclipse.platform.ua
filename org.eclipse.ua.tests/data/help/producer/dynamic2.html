<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<!--
   Test to make sure the dynamic content producer applies to XHTML inside
   a file with extension .html
-->

<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>dynamic2.html</title>
</head>
<body>

<h2>dynamic2.html</h2>
<p>
This page contains filtered content.
</p>
<p filter="os=invalid">
This paragraph, which has filter attribute "os=invalid" should always be filtered out.
</p>
<p filter="os!=invalid">
This paragraph, which has filter attribute "os!=invalid" should never be filtered out.
</p>
<p filter="plugin=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin=my.invalid.plugin.id.012345" should always be filtered out.
</p>
<p filter="plugin!=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin!=my.invalid.plugin.id.012345" should never be filtered out.
</p>
<p>
   <filter name="plugin" value="my.invalid.plugin.id.012345"/>
   This paragraph, which has filter element name="plugin", value="my.invalid.plugin.id.012345",
   should always be filtered out.
</p>
<p>
   <filter name="plugin" value="!my.invalid.plugin.id.012345"/>
   This paragraph, which has filter element name="plugin", value="!my.invalid.plugin.id.012345",
   should never be filtered out.
</p>

<include path="org.eclipse.ua.tests/data/help/producer/include2.html/my.paragraph.id"/>

</body>
</html>
