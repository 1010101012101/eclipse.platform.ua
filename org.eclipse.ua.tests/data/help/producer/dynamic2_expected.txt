<!--
   Test to make sure the dynamic content producer applies to XHTML inside
   a file with extension .html
--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

<title>dynamic2.html</title>
</head>
<body>

<h2>dynamic2.html</h2>
<p>
This page contains filtered content.
</p>

<p filter="os!=invalid">
This paragraph, which has filter attribute "os!=invalid" should never be filtered out.
</p>

<p filter="plugin!=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin!=my.invalid.plugin.id.012345" should never be filtered out.
</p>

<p>
   <filter name="plugin" value="!my.invalid.plugin.id.012345" />
   This paragraph, which has filter element name="plugin", value="!my.invalid.plugin.id.012345",
   should never be filtered out.
</p>

<p id="my.paragraph.id">
   This paragraph comes from <code>include2.html</code>.
</p>

</body>
</html>